flink-table/flink-table-common/src/main/java/org/apache/flink/table/catalog/CatalogPropertiesUtil.java

```diff
@@ -79,6 +80,10 @@
                 properties.put(COMMENT, comment);
             }
 
+            if (resolvedTable.getTableKind() == TableKind.MANAGED) {
```

Comment:
why do we need this? can't we derive the "managed" property from the options? looking for a missing `connector` should already be enough? I think we can assume that `connector.type` is not used by now.